Update do.m3u

Carlinhos027 opened the pull request:
Add new channels.

Closes #11158, Closes #11159

Carlinhos027 committed:
Update do.m3u

Add new channels.

Closes #11158, Closes #11159

LaneSh4d0w commented:
You can take a check up the Discord if you think you find other Dominican Republic channels to add :)

BellezaEmporium committed:
Merge branch 'master' into patch-30

BellezaEmporium committed:
Update do.m3u

BellezaEmporium committed:
Update do.m3u

telesistema 11 https://www.dailymotion.com/player/metadata/video/x80ac48?embedder=https%3A%2F%2Ftelesistema11.com.do
teleantillas https://www.dailymotion.com/player/metadata/video/x809r6i?embedder=https%3A%2F%2Fteleantillas.com.do
coral https://www.dailymotion.com/player/metadata/video/x84nyz2?embedder=https%3A%2F%2Fteleantillas.com.do

BellezaEmporium committed:
Merge branch 'master' into patch-30

Carlinhos027 committed:
Merge branch 'master' into patch-30

BellezaEmporium committed:
Merge branch 'master' into patch-30

LaneSh4d0w commented:
> Please add new IDs for: Telenovisa43.do TeleCaribe.do TelevisiondelEste.do VegavisionCanal18.do

Arhey confirmed to me that those were already available.

BellezaEmporium committed:
Merge branch 'master' into patch-30

BellezaEmporium committed:
Merge branch 'master' into patch-30
